api.Dockerfile

```diff
@@ -4,12 +4,13 @@ FROM node:lts-alpine AS builder
 WORKDIR /app
 ADD package.json yarn.lock ./
 ADD ./apis/core/package.json ./apis/core/
+ADD ./packages/core/package.json ./packages/core/
 
 # for every new package foo add
 #ADD ./packages/foo/package.json ./packages/foo/
 
 # install and build backend
-RUN yarn install --ci
+RUN yarn install --frozen-lockfile
```

Comment:
I figured that `--ci` is not a valid and yarn is not nice enough to point that out 🙄